src/Promitor.Agents.Scraper/Discovery/StubResourceDiscoveryRepository.cs

```diff
@@ -8,17 +8,17 @@ namespace Promitor.Agents.Scraper.Discovery
 {
     public class StubResourceDiscoveryRepository : IResourceDiscoveryRepository
     {
-        private static readonly List<AzureResourceDefinition> emptyResourceDefinitions = new();
-        private static readonly AgentHealthReport healthReport = new();
+        private static readonly List<AzureResourceDefinition> EmptyResourceDefinitions = new();
```

Comment:
Let's just revert to the old name if this is not related to your change, which applies to all these

Reply:
reverted